tensorflow/lite/tools/make/download_dependencies.sh

```diff
@@ -37,6 +37,7 @@ NEON_2_SSE_URL="https://github.com/intel/ARM_NEON_2_x86_SSE/archive/master.zip"
 FARMHASH_URL="http://mirror.tensorflow.org/github.com/google/farmhash/archive/816a4ae622e964763ca0862d9dbd19324a1eaf45.tar.gz"
 FLATBUFFERS_URL="http://mirror.tensorflow.org/github.com/google/flatbuffers/archive/v1.11.0.tar.gz"
 FFT2D_URL="http://mirror.tensorflow.org/www.kurims.kyoto-u.ac.jp/~ooura/fft2d.tgz"
+FP16_URL="https://github.com/Maratyszcza/FP16/archive/master.zip"
```

Comment:
Could you point to a specific commit ID or tag?

Reply:
AFAIK, fp16 provide only header file. I'll fix the `master.zip` to specific commit ID.

Reply:
BTW, intel/ARM_NEON_2_x86_SSE is used with master.zip

https://github.com/tensorflow/tensorflow/blob/4247d8e498d047dcf9a718638a931d68591d8f91/tensorflow/lite/tools/make/download_dependencies.sh#L36

Reply:
I'm sorry if I missed something obvious and asking naive question... 

It seems the current Makefile doesn't compile `delegates/egl/*` at all. 
So how does this change affect anything? Do you need to make a local change on `Makefile` to include `egl/` directory?

Reply:
Yes, but tensorflow team have plan to deprecate Makefile.

Reply:
@mattn I'm still not quite following the rationale for this CL. You say that it fixes the gpu/ build for the Makefile, but you're not actually compiling anything in gpu/. If you can verify a Makefile build for the rpi 4, we would be willing to consider such a pull request. However, without actually pulling in gpu/ to the Makefile, I don't think it's worth pulling in the fp16 library (unless the current standard TFLite Makefile build is broken, without the GPU code).

Reply:
If you allow me to put new Makefile on gpu directory, I'll add it. But you have plan to deprecate Makefiles.